AATAMS/AATAMS_acoustic/check_csv.py

```diff
@@ -0,0 +1,22 @@
+#! /usr/bin/env python
+"""
+Simple check (number of columns) of csv file format
```

Comment:
Since i list the column names i'm expecting i should probably check for their presenceI'll do the  modification
thanks @mhidas